frontend/src/app/main/ui/workspace/viewport/outline.cljs

```diff
@@ -78,19 +86,16 @@
    ::mf/wrap [#(mf/memo' % (mf/check-props ["shapes" "zoom" "modifiers"]))]}
   [props]
 
-  (let [shapes (obj/get props "shapes")
-        zoom   (obj/get props "zoom")
-        modifiers   (obj/get props "modifiers")
-        color  (if (or (> (count shapes) 1) (nil? (:shape-ref (first shapes))))
```

Comment:
We don't need the count of the shapes anymore? Only if it is in a component?

Reply:
The count was used only to select the color: blue if only 1 shape selected and it's a component; green elsewise.

But now the color is selected individually for each shape. So the count is not necessary.